src/components/ReviewList.jsx

```diff
@@ -44,6 +55,18 @@ export const ReviewList = ({ isLoading, setIsLoading }) => {
         <span> | </span>
         <Link to="/reviews/categories/engine-building">Engine Building</Link>
       </div>
+      <select onChange={handleQueryChange}>
```

Comment:
I'd look at fixing the default values - it would be better if it displayed whats currently selected/sorted by. 
At the moment, when you click, it has ascending clicked, the descending has actually been applied. Means when you click descending, nothing happens. You also have to click them twice to get them to appear in the box. 
This is the same for sort_by - i think the default is created_at, but it currently has the top one ticked.